blog/test_app.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+from webtest import TestApp
+from google.appengine.ext import testbed
+
+from bz_blog.wsgi import application
+
+
+class HomepageTest(unittest.TestCase):
+    def setUp(self):
+        self.testbed = testbed.Testbed()
```

Comment:
No need to init self.testbed as it is iniitalled in unittest.TestCase.

Use super(DatastoreTestCase, self).setUp()

Also no need to call self.testbed.deactivate() as it will be called from base class.